test/Nancy.Validation.FluentValidation.Tests.MSBuild/Nancy.Validation.FluentValidation.Tests.csproj

```diff
@@ -91,9 +91,9 @@
       <HintPath>..\..\packages\FakeItEasy.2.0.0-beta011\lib\net40\FakeItEasy.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="FluentValidation, Version=3.4.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <SpecificVersion>False</SpecificVersion>
-      <HintPath>..\..\packages\FluentValidation.3.4.0.0\lib\Net40\FluentValidation.dll</HintPath>
+    <Reference Include="FluentValidation, Version=6.3.4.0, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\FluentValidation.6.3.4-alpha\lib\Net45\FluentValidation.dll</HintPath>
+      <Private>True</Private>

```

Comment:
https://github.com/fsprojects/Paket/issues/566

On 1 June 2016 at 17:46, Andreas Håkansson <notifications@github.com> wrote:

> In
> test/Nancy.Validation.FluentValidation.Tests.MSBuild/Nancy.Validation.FluentValidation.Tests.csproj
> <https://github.com/NancyFx/Nancy/pull/2481#discussion_r65398632>:
>
> > @@ -91,9 +91,9 @@
> >        <HintPath>..\..\packages\FakeItEasy.2.0.0-beta011\lib\net40\FakeItEasy.dll</HintPath>
> >        <Private>True</Private>
> >      </Reference>
> > -    <Reference Include="FluentValidation, Version=3.4.0.0, Culture=neutral, processorArchitecture=MSIL">
> > -      <SpecificVersion>False</SpecificVersion>
> > -      <HintPath>..\..\packages\FluentValidation.3.4.0.0\lib\Net40\FluentValidation.dll</HintPath>
> > +    <Reference Include="FluentValidation, Version=6.3.4.0, Culture=neutral, processorArchitecture=MSIL">
> > +      <HintPath>..\..\packages\FluentValidation.6.3.4-alpha\lib\Net45\FluentValidation.dll</HintPath>
> > +      <Private>True</Private>
>
> private again
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NancyFx/Nancy/pull/2481/files/b61d92e51921fbf64f610ccca50a239916420d68#r65398632>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAGapplwbXJwguSmmowvn53nAGQRRwhEks5qHbdygaJpZM4IroZt>
> .
>